wolfssh/error.h

```diff
@@ -105,7 +105,8 @@ enum WS_ErrorCodes {
     WS_CLOSE_FILE_E         = -1065, /* Unable to close local file */
     WS_PUBKEY_REJECTED_E    = -1066, /* Server public key rejected */
     WS_EXTDATA              = -1067, /* Extended Data available to be read */
-
+    WS_USER_AUTH_E          = -1068, /* User authentication error */
+    
     WS_LAST_E               = -1067  /* Update this to indicate last error */
```

Comment:
This needs to be -1068.

Reply:
This also requires a new error string to be added to GetErrorString() in internal.c.

Reply:
Sorry. It's fixed.